### DISCUSSION TITLE
Why are there no 3.1 nuget packages?

### DISCUSSION THREAD

**Discussion by bpaczkowski** [dotnet/aspnetcore#29786] [General] [answered]
FrameworkReference seems to be the only way to reference asp.net core at the moment. Why were Nuget packages removed? They allowed more granular referencing of required functionality. For example a project responsible only for one aspect of the api could only reference that part of the asp.net core nuget suite. Now it is required to reference the entire asp.net core namespace even if only one type is used. I am curious what was the rationale behind this decision?

**Comment by davidfowl** [comment, marked as answer]
Start here https://github.com/aspnet/Announcements/issues/324.
